### ISSUE TITLE
Overriding Server.Project parameter not working

### ISSUE THREAD

**Issue by muttonchop** [reportportal/agent-net-vstest#16]
# Problem
When I am trying to override the project for a specific test launch, it does not use that overridden value and defaults to whatever is in the config file of the project.

# Setup
Given the following config file:
```json
 {
  "$schema": "https://raw.githubusercontent.com/reportportal/agent-net-vstest/master/ReportPortal.VSTest.TestLogger/ReportPortal.config.schema",
  "enabled": true,
  "server": {
    "url": "https://myReportPortalUri/api/v1/",
    "project": "default_test",
    "authentication": {
      "uuid": ""
    }
  },
  "launch": {
    "name": "Default",
    "description": "Default ReportPortal config",
    "debugMode": true,
    "tags": [ "default" ]
  }
}
```
Command: `dotnet vstest MyTestProject.dll --logger:ReportPortal;Server.Project=different-project`
## Expected:
Should report to the ReportPortal project with the name `different-project`

## Actual:
Reports results to the `default_test` project

# Additional Comments
`Server.Tags`, `Launch.Name`, and `Launch.Description` do override correctly. Problem persists in`v1.3.0` and `v2.0.0-beta.3`

**Comment by nvborisenko** [comment]
Reproduced. Are you OK to see the fix in beta version of agent?

**Comment by muttonchop** [comment]
Yes, I am very ok with that.

**Comment by nvborisenko** [comment]
The latest codebase already contains the fix, so releasing it as v2.0.0-beta.4.

@muttonchop you might be also interesting in parameters overriding via environment variables: https://github.com/reportportal/agent-net-vstest/tree/releases/2.0.0#environment-variables

**Comment by muttonchop** [comment]
Confirmed that this is fixed in `2.0.0-beta.4`  I will close this issue.
